src/procedural.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace procedural;
```

Comment:
Addressed with 31d0655506f5e0f50f2f7c4506b51193a09167dd. Totally forgot to `[ci skip]`. Derp.